.github/release/standard/Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
 RUN apt-get update
-RUN apt-get --yes install git
+RUN apt-get --yes install git gcc g++
```

Comment:
The point of shipping `wetterdienst-standard` is to provide a distribution image which is smaller than `wetterdienst-full` (1.19GB vs. 1.67GB) [^1]. So `wetterdienst-standard` has to ship without GDAL.

[^1]: I actually wonder if `wetterdienst-standard` could be made smaller ;].

Reply:
Yes it can be made smaller by putting everything into one layer and directly removing the cache:
```
RUN apt-get update &&
         apt-get --yes install git [gcc g++] &&
         rm -rf /var/lib/apt/lists/*
```

Reply:
`pip install --no-cache-dir` might also help

Reply:
We could also uninstall apt packages that are not needed at the end (like git)

Reply:
Sweet. Feel free to submit a corresponding patch which improves the situation.

Reply:
We might also use the buildx github action which is able to cache layers so it does not need to build every layer every time we change something small. I will have a look into it and provide my changes in a new PR

Reply:
@amotl 
Just tried the changes with removing git, gcc and g++ as well as using `no-cache-dir`.
When I build docker with `--platform linux/amd64` it builds fine. But using `arm64` it crashes because gcc is not installed. Seems there are no WHL arm64 files and it tries to build the package `numcodecs` from source. I tried both Python 3.8 and 3.9. I also tried to upgrade pip, wheel and setuptool but it does not help.

Reply:
> [...] using arm64 it crashes [...]

I see. Thanks for clarifying. I think Wetterdienst is not in ARM territory yet and we should address this separately. As far as Docker is concerned, I am recommending the `arm64v8/python:3.9-slim` Docker image.